### PR TITLE
Removing navbar, breadcrumbs, search icon from print

### LINKED ISSUE #1289: Print CSS
ToDo:

- [ ] do some research on Print CSS

- Show basic header
- Hide Navigation and everything that does not make much sense for a print out (e.g. social media buttons)

Resources:

- https://alistapart.com/article/goingtoprint/
- https://www.smashingmagazine.com/2011/11/how-to-set-up-a-print-style-sheet/

### PR DESCRIPTION
**Pull request by iFlameing**
Fixes #1289

### CODE REVIEW

**Comment by tisto** [comment]
[Welcome to Volto!.pdf](https://github.com/plone/volto/files/4504406/Welcome.to.Volto.pdf)

@iFlameing looks nice. I'd remove the footer as well. What do you think @sneridagh?

**Comment by sneridagh** [comment]
Yeah, same here. Sorry to be nit-picky, but the logo has some left margin. Please, try to make it at the same level of the text.

**Comment by iFlameing** [comment]
@tisto @sneridagh fixed both removed the footer and also align the logo with text :)

**Comment by tisto** [comment]
LGTM!

[Welcome to Volto!.pdf](https://github.com/plone/volto/files/4508247/Welcome.to.Volto.pdf)

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 ### Feature
 
+- Added Print CSS
```
Comment (tisto) [inline CHANGELOG.md:9]: @iFlameing please add your name here :)

**Pull request merged**